dali/pipeline/operators/displacement/rotate.h

```diff
@@ -33,9 +33,9 @@ class RotateAugment : public WarpAffineAugment {
     float angle = spec.GetArgument<float>("angle", ws, index);
     float angle_rad = angle * (M_PI / 180.0);
     p->matrix[0] = std::cos(angle_rad);
-    p->matrix[1] = std::sin(angle_rad);
```

Comment:
isn't this changing the behavior?

Reply:
Yes it is, to be aligned with all other libs (OCV or numpy) which rotates counterclockwise.